cpp/ql/src/experimental/Security/CWE/CWE-409-DecompressionBomb/Bzip2.qll

```diff
@@ -0,0 +1,50 @@
+/**
+ * https://www.sourceware.org/bzip2/manual/manual.html
+ */
+
+import cpp
+import semmle.code.cpp.ir.dataflow.TaintTracking
+import semmle.code.cpp.security.FlowSources
+import semmle.code.cpp.commons.File
+
+/**
+ * A `bz_stream` Variable as a Flow source
+ */
+class BzStreamVar extends VariableAccess {
+  BzStreamVar() { this.getType().hasName("bz_stream") }
+}
+
+/**
+ * A `BZFILE` Variable as a Flow source
+ */
+class BzFileVar extends VariableAccess {
+  BzFileVar() { this.getType().hasName("BZFILE") }
+}
```

Comment:
Classes like these that mention flow source in their comments seem to be unused. Unless I missed something, please delete them.